### PR TITLE
merge sort complete

### PR DESCRIPTION
**Pull request by jvmolu**
Anmol Singhal
11940140
anmolsinghal@iitbhilai.ac.in

### CODE REVIEW

File: Sorting/merge.cpp [hunk 1]
```diff
@@ -5,6 +5,72 @@
  *
  * Inputs: null
  */
-void Sort::MergeSort(){
-    std::cout<<"[TODO @ Sorting/merge.cpp] MergeSort() needs to be implemented.\n";
-}
+std::vector<int> arr;
```
Comment (yatendra1999) [inline Sorting/merge.cpp]: This should be avoided. Merge sort is a function from a class. It's okay to use a function from another scope(namespace or such) but using a global array which is not a part of that class should be avoided.

**Review by yatendra1999 — CHANGES_REQUESTED** [review]

File: Sorting/merge.cpp (hunk 1 above)
Comment (jvmolu) [reply Sorting/merge.cpp]: @yatendra1999  Done.

**Comment by shashwatj07** [comment]
@yatendra1999 ping

**Review by shashwatj07 — APPROVED** [review]
Looks good.

**Review by yatendra1999 — APPROVED** [review]

**Pull request merged**